src/components/SearchResults/UserInfo/UserInfo.js

```diff
@@ -1,17 +1,29 @@
 import styles from "./UserInfo.module.css";
 
-const UserInfo = ({ userInfo: { image, name, handle, bio, date } }) => {
+const UserInfo = ({
+  userInfo: { avatar_url, name, login, bio, created_at },
+}) => {
   return (
     <div className={styles.userInfo}>
-      <img className={styles.userImage} src={image} alt="User avatar" />
+      <img className={styles.userImage} src={avatar_url} alt="User avatar" />
```

Comment:
Currently when there's no user the page renders a broken image, we can deal with this by importing the octocat in this file and render it when there is no user and on initial render i.e 
```suggestion
      <img className={styles.userImage} src={avatar_url || octocat} alt="User avatar" />
```